packages/ens-controller/src/EnsController.test.ts

```diff
@@ -464,9 +462,8 @@ describe('EnsController', () => {
         provider: getProvider(),
         onNetworkStateChange: (listener) => {
           listener({
-            networkId: '1544',
             providerConfig: {
-              chainId: toHex(1),
+              chainId: toHex(0),
```

Comment:
Not sure I follow this change?

Reply:
`should return undefined when network is not ens supported`

previously it was using networkId to determine if ENS was supported or not. Now it uses chainId. I use a chainId of `0x0` as it isn't valid. Open to using some other value here. Ideas?

Reply:
ah ok makes sense